relay-server/src/services/processor.rs

```diff
@@ -1880,7 +1886,7 @@ impl EnvelopeProcessorService {
         &self,
         scoping: Scoping,
         buckets: Vec<Bucket>,
-        quotas: &[relay_quotas::Quota],
+        quotas: DynamicQuotas,
```

Comment:
micro nit, I like having the lifetime explicitly listed:
```suggestion
        quotas: DynamicQuotas<'_>,
```